deps/v8/src/runtime/runtime-i18n.cc

```diff
@@ -627,7 +627,7 @@ RUNTIME_FUNCTION(Runtime_CreateBreakIterator) {
 
   local_object->SetInternalField(0, reinterpret_cast<Smi*>(break_iterator));
   // Make sure that the pointer to adopted text is NULL.
```

Comment:
Note that the comments above applied to my original rejected patch to `0`, rather than upstream cherry pick, which went with `static_cast` instead of `reinterpret_cast` and `nullptr` for `0`.